server/config/config.go

```diff
@@ -156,6 +168,10 @@ func DefaultConfig() *Config {
 			Enable:  true,
 			Address: DefaultGRPCAddress,
 		},
+		StateSync: StateSyncConfig{
+			SnapshotInterval:   0,
```

Comment:
I think this should be a multiple of the default pruning strategy. It is best to default snapshots intervals because this will be something that is overlooked by most node operators. This could lead to very few nodes in a network able to provide snapshots.

Reply:
Yeah, I don't have a strong opinion here, so I'll let the SDK team decide on a default.

Reply:
We could pick a very relaxed option here so it happens very infrequently so we can get some testing. Like every 1k, 5k or 10k blocks

Reply:
Yeah, but if we do that then _all_ nodes in the network are on the same schedule. If there's a bug, _all_ nodes fail, and the entire network goes down.

Reply:
Fair point, let's keep at 0 then?

Reply:
How does it impact the storage and performance?

Is 0 === disabled?

Reply:
0 is disabled, yes. Taking a snapshot will cause a lot of IO, and the entire state at the snapshotted version (i.e. not including historical versions) will be stored in compressed form. If I remember correctly, when I tried this with the Cosmos Hub state back in February, it was less than 100 MB of data and took less than a minute to snapshot.